hotel/LICENSE/LICENSE

```diff
@@ -14,4 +14,3 @@
 
     You should have received a copy of the GNU Affero General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
```

Comment:
Shouldn't these files be removed altogether since we have a license file at the root? Let's see what the main contributors have to say.